GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -129,6 +129,9 @@ public sealed partial class FormCommit : GitModuleForm
         private readonly TranslationString _commitCommitterInfo = new TranslationString("Committer");
         private readonly TranslationString _commitCommitterToolTip = new TranslationString("Click to change committer information.");
 
+        private readonly TranslationString _modifyCommitMessageButtonToolTip
+            = new TranslationString("Enable the modification of this special commit message (Do it at your own risk!)");
```

Comment:
```suggestion
            = new TranslationString("If you change the commit message git will treat this commit as a ordinary commit (i.e. it may no longer be a fixup or an autosquash commit)");
```

I'm not sure if newline is rendered in tooltips, if it is - perhaps, add a newline before the left paren.